ElectronClient/gui/Header/Header.jsx

```diff
@@ -247,13 +256,35 @@ class HeaderComponent extends React.Component {
 			</a>
 		);
 
+
+		const fuzzyToggle = (
+			<div key={key + state.fuzzy.toString()}>
+				<div>
+					<input
+						id={`fuzzy_checkbox_${key}`}
+						type="checkbox"
+						checked={state.fuzzy}
+						onChange={this.toggleFuzzyValue}
+					/>
+					<label
+						onClick={this.toggleFuzzyValue}
```

Comment:
Please refactor so that you don't need to handle the even twice. I think if you wrap the input inside the label, or set the "for" attribute on the label you won't need this duplicate even handling.